src/projects/Browser.svelte

```diff
@@ -0,0 +1,192 @@
+<script lang="ts">
+  import { Project } from "../api/types/project";
```

Comment:
Need a `type` keyword here:

```suggestion
  import type { Project } from "../api/types/project";
```